### DISCUSSION TITLE
Pre-filtering in a Mastodon instance

### DISCUSSION THREAD

**Discussion by danielhuebert** [mastodon/mastodon#26092] [Q&A]
Hello all,

I'm having a bit of a challenge with server-side filtering right now. As I understand it, any user can filter specific keywords, other users or Mastodon instances for themselves. However, I can't currently find a setting to filter specific keywords in advance for an entire Mastodon instance so that individual users don't have to do this filtering themselves. Is this even possible with Mastodon? If so, where can I set this "pre-filtering" for an entire Mastodon instance?

Many thanks & greetings
Daniel

**Comment by colinstu12** [comment]
Issue open for this too #29256
